### PR TITLE
Add jenkins to marketplace

### PR DESCRIPTION
**Pull request by ruanbekker**
I used architecture as the category, as Im not too sure if there will be plans to use more categories as the time goes

### CODE REVIEW

File: jenkins/post_install.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```
+$ kubectl get pods --selector app=jenkins
+```
+
+Then exec into the container and retrieve the initial password:
+
+```
+$ kubectl exec -it jenkins-xxxxxxxxx-xxxxx -- cat /var/jenkins_home/secrets/initialAdminPassword
+423d1248daee49b8949a73abe52b3a43
+```
```
Comment (andyjeffries) [inline jenkins/post_install.md]: My only initial comment is, is it not possible to specify the initialAdminPassword from within a configmap or something or an ENV variable? Then we can randomly generate one using the `CIVO:ALPHANUMERIC(x)` value and have it presented in the Web UI and CLI? Like https://github.com/civo/kubernetes-marketplace/blob/master/postgresql/manifest.yaml#L29

Comment (ruanbekker) [reply jenkins/post_install.md]: Good point. Let me have a look into it.

**Comment by ruanbekker** [comment]
Changed upstream to bitnami as the default jenkins image does not allow the change of their initial admin password.

Chose to go with bitnami rather than my own custom one, as bitnami closely tracks upstream source changes and promptly publishes new versions of this image using their automated systems.

File: jenkins/post_install.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+By default external access to the Jenkins UI port isn't available. This is easily changed by applying the following YAML to your cluster with `kubectl apply -f jenkins-service.yaml` (or whatever you call the file containing the contents below):
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: jenkins-service
+spec:
+  type: LoadBalancer
```
[the remark this answers is not in the document]
Comment (ruanbekker) [reply jenkins/post_install.md]: Good point. I will look into this tomorrow

**Comment by ruanbekker** [comment]
- tested persistence with longhorn and works as expected
- added traefik ingress steps in post install

**Comment by andyjeffries** [comment]
Merging this and will be live on the site soon. We'll keep it in Architecture for now, but it may move to a new tab soon.

**Pull request merged**